src/core/value.rs

```diff
@@ -127,6 +131,36 @@ impl Value {
         )
     }
 
+    /// The notation X^<2 is notation for the type (S X)
+    /// The notation X^<(2*n) is notation for the type S (X^n) * X^<n
+    /// Cannot represent >= 2**16 bytes 0 <= n < 16.
+    ///
+    /// # Panics:
+    ///
+    /// Panics if the length of the slice is >= 2^(n + 1) bytes
```

Comment:
In 336740d2067f77f868d02823e799a448f811c139:

This doccomment doesn't say anything about the function that it's documenting, and it doesn't even use the notation that it defines!

Reply:
That notation is well defined in C simplicity and simplicity in general. But I agree, I should use the notation here instead of copy-pasting from C implementation.